org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLHighlighting.java

```diff
@@ -113,11 +113,11 @@ private static boolean isBeforeOrEqual(Position pos1, Position pos2) {
 				|| (pos1.getLine() == pos2.getLine() && pos1.getCharacter() <= pos2.getCharacter());
 	}
 
-	private static boolean covers(Range range, Position position) {
+	public static boolean covers(Range range, Position position) {

```

Comment:
I think it should be better to move this static method to an utility class like XMLPositionUtility